openedx/adg/lms/tests/constants.py

```diff
@@ -0,0 +1,4 @@
+""" Constants for ADG LMS tests """
+
+FIRST_NAME = 'test_first_name'
+FIRST_PART_OF_FULL_NAME = 'Test'
```

Comment:
Here, you can use:

FIRST_NAME = 'test_first_name'
LAST_NAME = 'test_last_name'

See my next comment for more clarity

Reply:
I've used the last_part_of_first_name in the same formatter, as it would be used just at one place, so we should define a constant for it.